src/Props/PreviousHackathonProps/props.ts

```diff
@@ -0,0 +1,33 @@
+//import { PreviousHackathonData } from "../../components/PreviousHackathons"
```

Comment:
It would be great to remove these unnecessary comments left over from the refactor.